snovault/tests/test_storage.py

```diff
@@ -143,12 +144,14 @@ def test_purge_uuid(session, storage):
     assert current.sid == propsheet.sid
     assert current.rid == resource.rid
 
+    assert len(storage.revision_history(uuid=str(resource.rid))) > 0
```

Comment:
If you're contrasting with the test 7 lines hence, I'd write this test as (untested):
```suggestion
    assert storage.revision_history(uuid=str(resource.rid)) != []
```
Alternatively (xor), you could change the other test to be `== 0`, I suppose. But I think this is good enough.